src/framework/components/particlesystem/particlesystem_system.js

```diff
@@ -197,10 +197,35 @@ pc.extend(pc, function() {
             }, {
                 name: "spawnBounds",
                 displayName: "Spawn Bounds",
-                description: "Defines an AABB in which particles are allowed to spawn",
+                description: "Defines an AABB/radius in which particles are allowed to spawn",
```

Comment:
Only X component is used for radius. I thought of different properties, but it's a bit of a pain to make it for each shape. Imagine cone for example: it has 2 radiuses and length. Fits perfectly into vec3 again. So are you sure we'll need 2 vec3s and 1 float instead just 1 vec3?